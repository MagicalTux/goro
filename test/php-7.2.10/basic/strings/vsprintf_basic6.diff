     *** Testing vsprintf() : basic functionality - using exponential format ***
002- string(11) "1.000000e+3"
003- string(23) "1.000000e+3 2.000000e+3"
004- string(35) "1.000000e+3 2.000000e+3 3.000000e+3"
002+ string(20) "1.00000000000000e+03"
003+ string(41) "1.00000000000000e+03 2.00000000000000e+03"
004+ string(62) "1.00000000000000e+03 2.00000000000000e+03 3.00000000000000e+03"
     Done
