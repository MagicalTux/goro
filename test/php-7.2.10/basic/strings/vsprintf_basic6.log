
---- EXPECTED OUTPUT
*** Testing vsprintf() : basic functionality - using exponential format ***
string(11) "1.000000e+3"
string(23) "1.000000e+3 2.000000e+3"
string(35) "1.000000e+3 2.000000e+3 3.000000e+3"
Done
---- ACTUAL OUTPUT
*** Testing vsprintf() : basic functionality - using exponential format ***
string(20) "1.00000000000000e+03"
string(41) "1.00000000000000e+03 2.00000000000000e+03"
string(62) "1.00000000000000e+03 2.00000000000000e+03 3.00000000000000e+03"
Done
---- FAILED
