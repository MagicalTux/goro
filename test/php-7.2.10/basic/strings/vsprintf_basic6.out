*** Testing vsprintf() : basic functionality - using exponential format ***
string(20) "1.00000000000000e+03"
string(41) "1.00000000000000e+03 2.00000000000000e+03"
string(62) "1.00000000000000e+03 2.00000000000000e+03 3.00000000000000e+03"
Done